microservices/nris_api/backend/run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -exv -o pipefail
+
+coverage erase
+coverage run --branch --source=app -m pytest
+coverage xml -i
+
+/opt/app-root/sonar-scanner-cli/bin/sonar-scanner
```

Comment:
Does this actually run? I thought you didnt install sonar-scanner yet

Reply:
no it does not. building on it next PR.